go.mod

```diff
@@ -121,8 +121,8 @@ require (
 	github.com/gorilla/handlers v1.5.2 // indirect
 	github.com/gorilla/websocket v1.5.0 // indirect
 	github.com/grafana/otel-profiling-go v0.5.1 // indirect
-	github.com/grafana/pyroscope-go v1.1.1 // indirect
-	github.com/grafana/pyroscope-go/godeltaprof v0.1.6 // indirect
+	github.com/grafana/pyroscope-go v1.1.2 // indirect
+	github.com/grafana/pyroscope-go/godeltaprof v0.1.8 // indirect
```

Comment:
> how did you bump this indirect deps? 

Manually! 😂 No, really. Bumping non-major releases is cool due to backward-compatibility assumption in Go modules. 

Yeah, this can be also achieved via `replace` but I don't see a huge reason for that here, more noise than the help.

Reply:
Yeah, I can also send the same patch to `celestia-core` 🫡

Reply:
Done https://github.com/celestiaorg/celestia-core/pull/1470

Reply:
Sounds good, manual bump works for me.